lib/bookmarks-view.coffee

```diff
@@ -57,10 +57,13 @@ class BookmarksView extends SelectListView
 
   viewForItem: (bookmark) ->
     bookmarkRow = bookmark.marker.getStartPosition().row
```

Comment:
Now that there's `bookmarkEndRow`, I think it would be better to explicitly name this `bookmarkStartRow`.